UnityProject/Assets/Scripts/Player/PlayerScript.cs

```diff
@@ -78,6 +78,10 @@ public class PlayerScript : ManagedNetworkBehaviour, IMatrixRotation, IAdminInfo
 
 	public float RTT;
 
+	[SyncVar]
+	public bool RcsMode;
```

Comment:
this one will be serialized and appear in editor. I assume this shouldn't be settable in inspector, use either ``[HideInInspector]`` or ``[NonSerializable]`` depending on which is best for this case.

Reply:
humm, michalczemierowski is not really the best at adding this stuff?
should I merge it?

Reply:
Can you suggest and commit? I’m on my phone now. Then we can merge

Reply:
ok cool

Reply:
Sorry, I must have missed it